packages/wattpm/lib/commands/management.js

```diff
@@ -0,0 +1,237 @@
+import { RuntimeApiClient } from '@platformatic/control'
```

Comment:
Why have you reimplemented this? Everything was already present in https://github.com/platformatic/platformatic/blob/main/packages/control/control.js#L25-L34.

Reply:
I misunderstood the scope of the issue. Sorry for that.